resources/language/Swedish/strings.po

```diff
@@ -365,20 +365,20 @@ msgid "Unable to find Elementum binary[CR]Required platform:"
 msgstr "Det gick inte att hitta Elementum-binär[CR]Nödvändig plattform:"
 
 msgctxt "#30104"
-msgid "Insert file or URL"
-msgstr "Infoga fil eller webbadress"
+msgid "Insert file or "
```

Comment:
@nonew-star IMPORTANT: do not change `msgid`, otherwise the whole section (msgctxt+msgid+msgstr) will be replaced with default values from [messages.pot](https://github.com/elgatito/plugin.video.elementum/blob/master/resources/language/messages.pot) by [xgettext_merge.sh](https://github.com/elgatito/plugin.video.elementum/blob/master/scripts/xgettext_merge.sh) script, so your translation for that message will be lost.
other parts looks ok, but i suggest you to double check, in case if i missed something.

Reply:
Fixed